monster_module.py

```diff
@@ -1,16 +1,31 @@
 #where the monsters are
 from random import randint
 
+#GL = 43961
+GL = 9785
+BN = 9792
+DW = 9786
+
 npc_goblin = {
     "name": "Gilbert the Goblin",
-    "hit_points": 10,
+    "hit_points": 5,
+    "attack_power": 4,
+    "defense": 0,
+    "defense_scalar": .5,
+    "speed": 6,
+    "battle_decision": None,
+    "symbol": chr(GL)
+}
+
+npc_goblin_two = {
```

Comment:
all this data should be in a yml file. You don't need "different" goblins. At least not right now. But you could have them no problem in the yml file. Then your "event" in your map file, as alluded to above, will just reference these monsters.